src/views/main-page/MainPage.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import { MainPageTopBar } from '.';
+import { MainPageBasicInformation } from './basic-information';
```

Comment:
You can name it Topbar and BasicInformation and export in in core index.ts file and import in same line with topbar. Remember we using feature modules pattern. Never think like that - im using in other application component which is called Navbar. Doesnt matter . Modules are separate units.